faunadb-common/src/main/java/com/faunadb/common/models/request/RequestParameters.java

```diff
@@ -0,0 +1,51 @@
+package com.faunadb.common.models.request;
+
+import com.faunadb.common.models.tags.Tag;
+
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+public class RequestParameters {
```

Comment:
let's javadoc this class since its public.

Reply:
Added (for `Tag` as well)